app/src/pages/admin/projectsPage/projectsPage.jsx

```diff
@@ -1 +1,175 @@
-export const ProjectsPage = () => <h1>Projects page</h1>;
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { redirect as rfrRedirect } from 'redux-first-router';
+import { injectIntl, defineMessages, intlShape } from 'react-intl';
+import classNames from 'classnames/bind';
+import { PageLayout, PageHeader, PageSection } from 'layouts/pageLayout';
+import {
+  PROJECTS_PAGE,
+  PROJECT_DETAILS_PAGE,
+  projectIdSelector,
+  projectSectionSelector,
+} from 'controllers/pages';
+import { GhostButton } from 'components/buttons/ghostButton';
+import AddProjectIcon from './img/add-project-inline.svg';
+import ProjectUsersIcon from './img/project-users-inline.svg';
+import ProjectSettingsIcon from './img/project-settings-inline.svg';
+import ProjectEventsIcon from './img/project-events-inline.svg';
+
+import styles from './projectsPage.scss';
+
+const cx = classNames.bind(styles);
+
+const messages = defineMessages({
+  pageTitle: {
+    id: 'ProjectsPage.title',
+    defaultMessage: 'All projects',
+  },
+  settingsTitle: {
+    id: 'ProjectDetailsPageSettings.title',
+    defaultMessage: 'Settings',
+  },
+  membersTitle: {
+    id: 'ProjectDetailsPageMembers.title',
+    defaultMessage: 'Members',
+  },
+  eventsTitle: {
+    id: 'ProjectDetailsPageEvents.title',
+    defaultMessage: 'Events',
+  },
+});
+
+const HEADER_BUTTONS = [
+  {
+    key: 'members',
```

Comment:
May be we need to create `projectDetailsSections` in `common/constants` and hold these values here

Reply:
Done